ql/src/java/org/apache/hadoop/hive/ql/ddl/table/storage/skewed/AlterTableSkewedByDesc.java

```diff
@@ -70,6 +70,6 @@ public boolean isStoredAsDirectories() {
 
   @Override
   public boolean mayNeedWriteId() {
-    return false;
+    return true;
```

Comment:
AFAIK we do not support skewed ACID tables

Reply:
Right, was not aware of it. Flipping it back to false.